Profile UI

ChrisTHChan opened the pull request:
- Override default theme components

nicboy17 committed:
Profile UI

- Override default theme components

ChrisTHChan committed:
profile UI, +login and signup here too

nicboy17 reviewed (approved):
Looks good, matches the RFP pretty well. We're probably going to have to do some refactoring later, so we can reuse some of the styling and components.

ChrisTHChan committed:
post requests

committing code for post requests for /login and /signup, also some style changes to /profile. ticket 9

ChrisTHChan committed:
Merge branch 'master' into profile-page-#15

ChrisTHChan committed:
final commit
